cmd/pb-td-gen/linked_rpc.go

```diff
@@ -0,0 +1,40 @@
+package main
```

Comment:
I would use this file and traverse_linked_rpc.go to the package `pb-td-gen`
For me, the philosophy of separating them in two files was not explicitly clear.